packaging/windows/buildpackage.sh

```diff
@@ -90,6 +95,12 @@ function build_platform()
 		exit 1
 	fi
 
+	echo "Packaging zip archive ($1)"
+	pushd "${BUILTDIR}" > /dev/null
+	zip "OpenRA-${TAG}-${1}-winportable.zip" -r -9 * .[^.]* --quiet
```

Comment:
Why is the `.[^.]*` needed here? The SDK packaging uses `zip "${PACKAGING_INSTALLER_NAME}-${TAG}-${1}-winportable.zip" -r -9 * --quiet` without trouble (afaik).

Reply:
I forgot. It's been too long. Maybe to exclude the `.itch.toml` file?

Reply:
Right, that makes sense.

The .itch.toml is now added explicitly in `upload-itch.sh`, so can we please remove the `.[^.]*` here?

Reply:
Done.